lib/ansible/plugins/lookup/subelements.py

```diff
@@ -74,7 +74,7 @@
 
     - name: list groups for user that have them, dont error if they don't
```

Comment:
While you're in this file, can you fix the typo (dont => don't in line 75)?

Reply:
Done, I reworded it a little as well.